R/sensorreading-data.R

```diff
@@ -266,7 +266,7 @@ getSRSQualifiers <- function(inQualifiers) {
 
 getComments <- function(comments) {
   comm <- unlist(comments)
-  if (!isEmpty(comm)) {
+  if (!isEmptyOrBlank(comm) && length(comm)>0) {
```

Comment:
honestly, I would add this to the isEmptyOrBlank function. Replace 206 with `result <- (length(val) == 0 || isEmpty(val) || val=="")`